dask/utils.py

```diff
@@ -1837,7 +1840,7 @@ def key_split(s):
     try:
         words = s.split("-")
         if not words[0][0].isalpha():
-            result = words[0].strip("_'()\"")
+            result = words[0].split(",")[0].strip("_'()\"")
```

Comment:
This is very interesting. Can you add a test for this?

Reply:
I'm sure this is fine, I'm just surprised to see these implementations to drift. FWIW this change was introduced in https://github.com/dask/distributed/pull/1566

Reply:
Ok, just saw that we're executing doc tests.